packages/react-cs-renderer/src/ReactNativeCSTypes.js

```diff
@@ -5,19 +5,26 @@
  * LICENSE file in the root directory of this source tree.
  *
  * @flow
+ * @providesModule ReactNativeCSTypes
```

Comment:
![](https://i.giphy.com/E2d2tsgz7iHo4.gif)

Reply:
I guess we'll need to add it and `react-native-renderer/src/ReactNativeTypes.js` to [this list](https://github.com/facebook/react/blob/cc54b6f48a532bf2073e8f7d888c6a3fcf416b63/scripts/circleci/check_modules.sh#L6):

```
EXPECTED='scripts/rollup/header.js
packages/react-native-renderer/src/ReactNativeTypes.js
packages/react-cs-renderer/src/ReactNativeCSTypes.js'
```